### PR TITLE
Test clover suite

### PR DESCRIPTION
**Pull request by adamjarling**
## What does this do?

Pulls in the new Clover Release `@samvera/clover-iiif@latest`

## How to test
Fire up the app and view the Work page.  Note the Viewer, Metadata, and Related Items.  

Also view Collection pages or anywhere else a `Slider` component may be used.

### CODE REVIEW

**Comment by adamjarling** [comment]
@mathewjordan This one is now good to go with `latest` Clover release.   Tests passing.

File: components/Clover/SliderWrapper.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
+import Slider from "@samvera/clover-iiif/slider";
```
Comment (mathewjordan) [inline components/Clover/SliderWrapper.tsx:1]: Liking the moving of this to a `Clover` directory. Should we also move the **ViewerWrapper** as well at some point?

**Review by mathewjordan — APPROVED** [review]
Works like a dream.

File: components/Clover/SliderWrapper.tsx (hunk 1 above)
Comment (adamjarling) [reply components/Clover/SliderWrapper.tsx:1]: @mathewjordan Yes, good idea.  I'll do that before the squash.

**Pull request merged**